docs/sources/write/shortcodes/index.md

```diff
@@ -514,13 +514,18 @@ For more detailed instructions, refer to [Reuse shared content](https://grafana.
 | `lookup`      | Path to the included content relative to the root of the shared directory.                                                                                                                                                                                                                                                     | yes      |
 | `source`      | Name of the source content as shown on the website. For example, for https://grafana.com/docs/enterprise-metrics/ content, the _source_ is `enterprise-metrics`.                                                                                                                                                               | yes      |
 | `version`     | Version of the source content to include. For source content that doesn't have a version, use the empty string `""` as the value. This shortcode supports version substitution using values like `<GRAFANA_VERSION>`. To learn about version substitution, refer to [About version substitution](#about-version-substitution). | yes      |
-| `leveloffset` | Manipulates source content headings up to a maximum level of `h6`. Only positive offsets are currently supported. `leveloffset="+5"` ensures an `h1` in the source content is an `h6` in the destination content.                                                                                                              | no       |
+| `leveloffset` | Manipulates source content headings up to a maximum level of `h6`. Only positive offsets are supported. `leveloffset="+5"` ensures an `h1` in the source content is an `h6` in the destination content.                                                                                                                        | no       |
 
 {{< admonition type="note" >}}
 Hugo doesn't rebuild the destination file when a source file changes on disk.
 To trigger a rebuild after changes to a source file, perform a trivial change to the destination file and save that, too.
 {{< /admonition >}}
 
+### Guidance
+
+When the page with the `docs/shared` shortcode includes a shared page from the same project, you should use version substitution syntax for the `version` parameter.
```

Comment:
What do you do if it doesn't?

Reply:
If it doesn't, you should use the version of the docs that you want to include from. Usually I guess this would be latest. 

Do you think it would be useful to include that here too?

Reply:
My question was prompted by the conditions in the sentence... if there is a shared page in the same project then use version substitution.  OK, so then what happens if there's a shared page that's not in the same project?

I think it'd be useful to cover both scenarios... otherwise we leave the reader hanging, wondering what to do in the "else" case.

Does this apply to say... a topic in /agent/shared that's reused in/shared to Loki? You don't use AGENT_VERSION anymore? You use LATEST?

Reply:
To simplify the guidance, I've chosen to focus only on version substitution syntax in https://github.com/grafana/writers-toolkit/pull/639/files#r1602869550